engine/engine/content/builtins/input/default.gamepads

```diff
@@ -5559,3 +5559,34 @@ driver
     map { input: GAMEPAD_BACK type: GAMEPAD_TYPE_BUTTON index: 16 }
     map { input: GAMEPAD_GUIDE type: GAMEPAD_TYPE_BUTTON index: 19 }
 }
+
+driver
+{
+    device: "Microsoft X-Box 360 pad 0"
```

Comment:
Is this the reported name of the stem deck controllers?

Reply:
Yes, a weird name tbh:

https://forum.defold.com/t/steam-deck-input-solved/70881/33?u=britzl

Reply:
I assume it's done for compatibility reasons. Most games look for the 360 controller so that's why they spoof it.